recipes-extend/sd-creator-linux/files/sd_flash.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+
+function chelp() {
+	echo "------------------------------------------------------"
+	echo "Command format:"
+	echo "    sd_flash.sh <device> <location of the root filesystem image>"
+	echo "    Example: sd_flash.sh /dev/sda <path/to/your/images.tar.bz2>"
+	echo ""
+	echo "If no <location of the root filesystem image> is provided in the command,"
+	echo "the default location is set to ${IMAGE_BASENAME}-rzg2l-sbc.tar.bz2 inside any release package"
+	echo "------------------------------------------------------"
+}
+
+# Default core-image
+rootfs_image_path="../../../../target/images/rootfs/${IMAGE_BASENAME}-rzg2l-sbc.tar.bz2"
```

Comment:
Did you check the script and verified them? I don't think IMAGE_BASENAME make senses.